### PR TITLE
Implement view weekly and daily Timetable

### PR DESCRIPTION
**Pull request by LouisLouis19**
Implement #18 and #100

### CODE REVIEW

File: src/main/java/terminus/command/TimetableCommand.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            result.append(listDailySchedule(contentManager, today));
+            TerminusLogger.info(String.format("Successfully acquire %s's schedule for %s", moduleName, today));
+        }
+        index = 0;
+        if (result.toString().isEmpty()) {
+            return false;
+        } else {
+            return true;
+        }
+    }
```
Comment (3m0W33D) [inline src/main/java/terminus/command/TimetableCommand.java:97]: why are the method names static?

File: src/main/java/terminus/command/TimetableCommand.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    @Override
+    public String getHelpMessage() {
+        return Messages.MESSAGE_COMMAND_TIMETABLE;
+    }
+
+    /**
+     * Parses remaining arguments for the timetable command.
+     *
+     * @param arguments The string arguments to be parsed in to the respective fields.
+     * @throws InvalidArgumentException when arguments are invalid
```
Comment (Woolicious98) [inline src/main/java/terminus/command/TimetableCommand.java:44]: I think its missing the full stop "." at the end for the comments.

File: src/main/java/terminus/timetable/Timetable.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+     * Print empty message for empty user schedule.
+     *
+     * @param result The string containing the retrieved user schedule
+     */
+    public static void checkEmptySchedule(StringBuilder result) {
+        if (result.toString().isBlank()) {
+            TerminusLogger.info("There is no schedule in the user's timetable");
+            result.append(Messages.EMPTY_CONTENT_LIST_MESSAGE);
+        }
+    }
```
Comment (3m0W33D) [inline src/main/java/terminus/timetable/Timetable.java:88]: Please remove the static methods

**Review by 3m0W33D — COMMENTED** [review]
Remove the static methods

File: src/main/java/terminus/timetable/Timetable.java (hunk 3 above)
Comment (LouisLouis19) [reply src/main/java/terminus/timetable/Timetable.java]: okay removed!

**Review by kelvneo — APPROVED** [review]
LGTM

**Pull request merged**